packages/shared/src/hooks/useLeaveSquad.ts

```diff
@@ -20,9 +24,15 @@ type UseLeaveSquadProps = {
 };
 
 export const useLeaveSquad = ({ squad }: UseLeaveSquadProps): UseLeaveSquad => {
+  const queryClient = useQueryClient();
   const { logEvent } = useContext(LogContext);
   const { showPrompt } = usePrompt();
+  const { user } = useAuthContext();
   const { deleteSquad: deleteCachedSquad } = useBoot();
+  const contentPrefKey = generateQueryKey(RequestKey.ContentPreference, user, {
```

Comment:
Same here, don't think you'd need to invalidate, it should handle it well underwater-appwide already.